src/OpenSage.Game/Utilities/LanguageUtility.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using OpenSage.Data;
+
+namespace OpenSage.Utilities
+{
+    public static class LanguageUtility
+    {
+        private const string DefaultLanguage = "english";
+
+        /// <summary>
+        /// Detect the language for the current installation
+        /// </summary>
+        /// <param name="gameDefinition"></param>
+        /// <param name="rootDirectory"></param>
+        /// <returns>language as string</returns>
+        public static string ReadCurrentLanguage(IGameDefinition gameDefinition, string rootDirectory)
+        {
+            if (Environment.OSVersion.Platform == PlatformID.Win32Windows)
```

Comment:
Does this work? I think it should be Win32NT. Win32Windows was only for Windows95 and 98.